lib/LocalVerifiableCredentialStore.js

```diff
@@ -118,11 +140,12 @@ export class LocalVerifiableCredentialStore {
   async delete({id}) {
     try {
       const selector = {
-        'content.id': id,
         'meta.profileId': this.profileId,
+        'content.id': id
       };
       const {docs: [doc]} = await this.db.find({
         selector,
+        use_index: 'getDoc',
```

Comment:
If you don't specify an index, is it not smart -- can it not pick the right one?

Reply:
It is supposed to yes. I can do this, I just thought it will be a faster request if specified but maybe not. I can remove this since it seems to cause confusion.